<launch>
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
    <arg name="port" value="9090" />
  </include>
  <node pkg="web_interface" type="waypoints_node.py" name="web_interface_waypoint" />
  <node pkg="web_video_server" type="web_video_server" name="web_video_server">
    <param name="port" value="8000" />
  </node>
</launch>